tests/openwisp2/sample_users/models.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth.models import Group as AbstractGroup
+from django.core.validators import RegexValidator
+from django.db import models
+from organizations.abstract import (
+    AbstractOrganization,
+    AbstractOrganizationOwner,
+    AbstractOrganizationUser,
+)
+
+from openwisp_users.base.models import (
+    AbstractUser,
+    BaseGroup,
+    BaseOrganization,
+    BaseOrganizationOwner,
+    BaseOrganizationUser,
+)
+
+
+class User(AbstractUser):
+    social_security_number = models.CharField(
+        max_length=11,
+        null=True,
+        blank=True,
+        validators=[RegexValidator(r'^\d\d\d-\d\d-\d\d\d\d$')],
+    )
+
+    class Meta(AbstractUser.Meta):
+        abstract = False
+
+
+class Organization(BaseOrganization, AbstractOrganization):
+    pass
+
+
+class OrganizationUser(BaseOrganizationUser, AbstractOrganizationUser):
+    pass
+
+
+class OrganizationOwner(BaseOrganizationOwner, AbstractOrganizationOwner):
+    pass
+
+
+class Group(BaseGroup, AbstractGroup):
+    pass
```

Comment:
Shouldn't these contain following?
```python
class Meta:
   abstract = False
```
Like it has been done for User model.

Reply:
Actually, `abstract=True` is not required to be defined. Children of abstract base classes don't automatically become abstract classes themselves. 
But it is defined in `sample_users` of all other repositories. Should I remove it here?

Reply:
the correct form is:


```
class Meta(AbstractModel.Meta):
   abstract = False
```

it's done to inherit the model options, which otherwise do not get inherited (things like `verbose_name`, etc.).

Reply:
Done